packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -242,14 +259,14 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
                   ? onRenderTitle(selectedOptions, this._onRenderTitle)
                   : onRenderPlaceHolder(this.props, this._onRenderPlaceHolder)}
               </span>
-              <span className={css('ms-Dropdown-caretDownWrapper', styles.caretDownWrapper)}>
+              <span className={this._classNames.caretDownWrapper}>
                 {onRenderCaretDown(this.props, this._onRenderCaretDown)}
               </span>
             </div>
           )}
         </KeytipData>
         {isOpen && onRenderContainer(this.props, this._onRenderContainer)}
-        {errorMessage && <div className={css(styles.errorMessage)}>{errorMessage}</div>}
+        {errorMessage && errorMessage.length > 0 && <div className={this._classNames.errorMessage}>{errorMessage}</div>}
```

Comment:
a string with length 0 should already be falsey

Reply:
Preexisting code 
https://github.com/OfficeDev/office-ui-fabric-react/blob/729a6d562c40e418c7b4d5cdb2a8802a28feab9c/packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx#L230

but I thought the `> 0` is helpful clarity (since 0 being falsey is often a source of bugs) for very little extra bytes so I retained it...